src/main/java/Duke.java

```diff
@@ -1,10 +1,60 @@
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
+    	String lines = "--------------------------------------------"; 
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?"); 
+        System.out.println(lines);
+
+
+        String line;
+        Task[] tasks = new Task[100]; 
+        int items = 0; 
```

Comment:
Maybe numOfItems or itemCount will be a better choice for the variable name here